### PR TITLE
ELN Publishing (Frontend)

### PR DESCRIPTION
**Pull request by thomas-vu**
Screen Shot 2022-01-26 at 5 12 43 PM

### CODE REVIEW

File: components/CKEditor/SimpleEditor.js [hunk 1]
```diff
@@ -61,26 +68,33 @@ export default function SimpleEditor(props) {
...
+              data={initialData}
+              editor={isBalloonEditor ? SimpleBalloonEditor : SimpleEditor}
+              id={id}
+              onChange={(event, editor) => {
+                onChange(id, editor.getData());
+              }}
+              onReady={(editor) => {
+                if (readOnly) {
+                  editor.isReadOnly = true;
+                } else {
```
[the remark this answers is not in the document]
Comment (thomas-vu) [reply components/CKEditor/SimpleEditor.js:84]: This min-height is for the editor so it doesn't appear squished when there's no content.

Screen Shot 2022-02-21 at 10 19 39 AM

File: components/CKEditor/SimpleEditor.js [hunk 2]
```diff
@@ -39,7 +44,9 @@ export default function SimpleEditor(props) {
   useEffect(() => {
     editorRef.current = {
       CKEditor: require("@ckeditor/ckeditor5-react").CKEditor,
-      Editor: require("@thomasvu/ckeditor5-custom-build").SimpleEditor,
+      SimpleEditor: require("@thomasvu/ckeditor5-custom-build").SimpleEditor,
```
[the remark this answers is not in the document]
Comment (thomas-vu) [reply components/CKEditor/SimpleEditor.js:47]: For sure, I can set that up soon.

File: components/Modals/NotePublishModal.tsx [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+      .then(Helpers.checkStatus)
+      .then(Helpers.parseJSON)
+      .then((resp) => {
+        /* @ts-ignore */
+        let hubs = resp.results
+          .map((hub, index) => {
+            return {
+              ...hub,
+              value: hub.id,
+              label: hub.name.charAt(0).toUpperCase() + hub.name.slice(1),
```
[the remark this answers is not in the document]
Comment (thomas-vu) [reply components/Modals/NotePublishModal.tsx]: Good catch, we didn't really need this.

**Review by yattias — APPROVED** [review]
LGTM

**Pull request merged**